x-pack/platform/plugins/private/upgrade_assistant/public/application/app.tsx

```diff
@@ -121,7 +121,6 @@ const AppHandlingClusterUpgradeState: React.FunctionComponent = () => {
     <Routes>
       <Route exact path="/overview" component={Overview} />
       <Route exact path="/es_deprecations" component={EsDeprecations} />
-      <Route exact path="/es_deprecation_logs" component={EsDeprecationLogs} />
```

Comment:
nit: is there any chance we can make the flyout stateful through the url and then do a redirect from this old route into the flyout url? That way if for any reason users had the url bookmarked it will still work